services/minespace-web/src/components/dashboard/mine/overview/Overview.js

```diff
@@ -46,110 +46,119 @@ const getMineManager = (partyRelationships) => {
-        information is pulled from current Ministry resources. If anything is incorrect, please
-        notify one of the Ministry contacts.
-      </Typography.Paragraph>
-      <Descriptions column={2} colon={false}>
-        <Descriptions.Item span={2} label="Region">
-          {props.mineRegionHash[props.mine.mine_region] || Strings.UNKNOWN}
-        </Descriptions.Item>
-        <Descriptions.Item label="Latitude">
-          {(props.mine.mine_location && props.mine.mine_location.latitude) || Strings.UNKNOWN}
-        </Descriptions.Item>
-        <Descriptions.Item label="Longitude">
-          {(props.mine.mine_location && props.mine.mine_location.longitude) || Strings.UNKNOWN}
-        </Descriptions.Item>
-        <Descriptions.Item span={2} label="Operating Status">
-          {(props.mine.mine_status &&
-            props.mine.mine_status.length > 0 &&
-            props.mine.mine_status[0].status_labels.join(", ")) ||
-            Strings.UNKNOWN}
-        </Descriptions.Item>
-        <Descriptions.Item span={2} label="Commodity">
-          {props.transformedMineTypes &&
-          props.transformedMineTypes.mine_commodity_code &&
-          props.transformedMineTypes.mine_commodity_code.length > 0
-            ? props.transformedMineTypes.mine_commodity_code
-                .map((code) => props.mineCommodityOptionsHash[code])
-                .join(", ")
-            : Strings.UNKNOWN}
-        </Descriptions.Item>
-        <Descriptions.Item span={2} label="Disturbance">
-          {props.transformedMineTypes &&
-          props.transformedMineTypes.mine_disturbance_code &&
-          props.transformedMineTypes.mine_disturbance_code.length > 0
-            ? props.transformedMineTypes.mine_disturbance_code
-                .map((code) => props.mineDisturbanceOptionsHash[code])
-                .join(", ")
-            : Strings.UNKNOWN}
-        </Descriptions.Item>
-        <Descriptions.Item span={2} label="Active Permits">
-          {props.mine.mine_permit_numbers && props.mine.mine_permit_numbers.length > 0
-            ? props.mine.mine_permit_numbers.join(", ")
-            : Strings.NONE}
-        </Descriptions.Item>
-      </Descriptions>
-      <div className="padding-md--top padding-md--bottom">
-        <MineWorkInformation mineGuid={props.mine.mine_guid} />
-      </div>
-      <div className="padding-md--top padding-md--bottom">
-        <WorkerInfoEmployee mine={props.mine} />
-      </div>
-      <Row gutter={[16, 16]}>
-        <Col xl={11} xxl={11} md={24}>
-          <ContactCard
-            title="Mine Manager"
-            partyRelationship={getMineManager(props.partyRelationships)}
-            dateLabel="Mine Manager Since"
-          />
-        </Col>
+export const Overview = (props) => {
+  const { mine } = useContext(SidebarContext);
+  return (
+    <>
+      <Row>
+        <Typography.Title level={1}>Overview</Typography.Title>
       </Row>
```

Comment:
The only real change here is pulling out the header from the same row as everything else, and putting it on its own row up top. The rest is whitespace + changing `props.mine` --> `mine`